webplugin/js/app/zendesk-chat-service.js

```diff
@@ -7,6 +7,7 @@ function ZendeskChatService() {
     var ZENDESK_CHAT_SDK_KEY = "";
     var AGENT_INFO_MAP = {};
     var preChatLeadData = {};
+    var numberUpdatedInZopim = false;
```

Comment:
this will be false again in every page refresh and that means we will keep calling the API again?